src/main/java/ui/PanelMenuCreator.java

```diff
@@ -21,23 +21,17 @@
 
 public class PanelMenuCreator {
 
-    private static final Logger logger = LogManager.getLogger(MenuControl.class.getName());
-
-    public static final String MILESTONE_FILTER_NAME = "milestone:curr sort:status";
-    public static final String MILESTONE_PANEL_NAME = "Current Milestone";
-
-    public static final String ASSIGNEE_FILTER_NAME = "is:open ((is:issue assignee:me) OR (is:pr author:me))";
-    public static final String ASSIGNEE_PANEL_NAME = "Open issues and PR's";
+    private final Preferences prefs;
 
-    public static final String UPDATED_FILTER_NAME = "assignee:me updated:<48";
-    public static final String UPDATED_PANEL_NAME = "Recently Updated issues";
+    private static final Logger logger = LogManager.getLogger(MenuControl.class.getName());
 
     private final PanelControl panelControl;
     private final ScrollPane panelsScrollPane;
 
-    public PanelMenuCreator(PanelControl panelControl, ScrollPane panelsScrollPane) {
+    public PanelMenuCreator(PanelControl panelControl, ScrollPane panelsScrollPane, Preferences prefs) {
```

Comment:
Actually, if the last login username is all we need here, we can pass that instead of all of prefs. Unless you foresee us needing other fields?